frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/entity/AuthCodeRequest.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.di.authentication.frontendapi.entity;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+import uk.gov.di.authentication.shared.validation.Required;
+
+public class AuthCodeRequest {
+
+    @SerializedName("redirect-uri")
+    @Expose
+    @Required
+    private String redirectUri;
+
+    @SerializedName("state")
+    @Expose
+    @Required
+    private String state;
+
+    @SerializedName("requestedScopeClaims")
```

Comment:
Sorry to nitpick, but there was some discussion around the nature of what would be held in this field. For example, at first we weren't sure if we could hold a scope like `email` and have that unpacked into claims (being `email` and `email_verified`).Same for phone. After discussing with @JHjava , he said the authorize handler _would_ do this unpacking in future (I have made that assumption in the recent PR for the userinfo handler). Although there's no functional effect, I wonder if we might just call this (and any similar references) `claims` and not bring scopes into it?

Reply:
As the change is out of scope from what was initially required in the ticket, I will create another ticket and convert the field to List of string (claims) instead.

Reply:
I've created a subtask to rename that field: https://govukverify.atlassian.net/browse/AUT-1592